corehq/apps/locations/tests/util.py

```diff
@@ -62,27 +62,31 @@ def create_locations(locations, types, parent):
     return locations_dict
 
 
-def setup_locations_and_types(domain, location_types, locations):
+def setup_locations_and_types(domain, location_types, stock_tracking_types, locations):
```

Comment:
I considered that, but that makes it more difficult to use in tests, and that's really where I want the intent to be obvious.